README.md

```diff
@@ -66,7 +66,9 @@ Patches and issues welcome!
 
 ### Video chat ###
 
-[Complete video chat client (based on Google App Engine)](https://apprtc.appspot.com)
+[AppRTC video chat client](https://apprtc.appspot.com)
```

Comment:
Keep the "powered by Google App Engine" thing here - this is a continual source of confusion